src/styles/blocks/star.scss

```diff
@@ -0,0 +1,38 @@
+.stars {
+  padding-inline: 16px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  position: absolute;
+  top: 276px;
```

Comment:
Using absolute positioning for the stars component might cause layout issues on different screen sizes. Consider using a more flexible layout approach, such as flexbox or grid, to ensure responsiveness.